docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc

```diff
@@ -386,6 +386,186 @@ It is possible to use multiple expressions in the role definition.
 <3> This `/subject/user` endpoint requires an authenticated user that has been granted the role "User" through the use of the `@RolesAllowed("${customer:User}")` annotation, as we did not set the configuration property `customer`.
 <4> This `/subject/secured` endpoint requires an authenticated user that has been granted the role `User` in production but allows any authenticated user in development mode.
 
+=== Permission annotation
+
+Quarkus also provides the `io.quarkus.security.PermissionsAllowed` annotation that will permit any authenticated user with given permission to access the resource.
+The annotation is extension of the common security annotations and there is no relation to configuration permissions defined with the configuration property `quarkus.http.auth.permission`.
+
+.Example of endpoints secured with a `@PermissionsAllowed` annotation
+
+[source,java]
+----
+import io.quarkus.arc.Arc;
+import io.vertx.ext.web.RoutingContext;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.QueryParam;
+
+import io.quarkus.security.PermissionsAllowed;
+
+import java.security.Permission;
+import java.util.Collection;
+import java.util.Collections;
+
+@Path("/crud")
+public class CRUDResource {
+
+    @PermissionsAllowed("create") <1>
+    @PermissionsAllowed("update")
+    @POST
+    public String createOrUpdate() {
+        return "modified";
+    }
+
+    @PermissionsAllowed({"see:detail", "see:all", "admin"}) <2>
```

Comment:
What is `see:detail` or rather why such a namespaced/qualified permission is highlighted with `:` ? Is it just whatever the custom permission/autgmentor chooses to name a permission ?
Also a permission named `admin` appears to be a role name (I guess it is not but it might be a bit confusing)

Reply:
Also, why are there `permission name` and `actions` (not that you asked :-), just for context)? That's because `java.security.Permission` works with permission name and actions, while actions are optional. This fashion is followed by the annotation.

Reply:
Re-think it - `see` sounds like an action. I'll adjust it.

Reply:
In terms of meaning (dictionary wise), permission is action, I'm not sure whether the examples are wrong. `admin` must go though.

Reply:
Done.

Reply:
@michalvavrik Thanks, thanks for the clarification